RELEASE.md

```diff
@@ -142,6 +142,8 @@
     *   Removed experimental tf.data API for map vectorization:
         *   `tf.data.experimental.OptimizationOptions.map_vectorization`
         *   `tf.data.experimental.MapVectorizationOptions.*`
+    *   Promoting `tf.data.experimental.OptimizationOptions` API to
```

Comment:
we should also document the introducing of `tf.data.AutotuneOptions`

Reply:
addressed in https://github.com/tensorflow/tensorflow/pull/50291